### ISSUE TITLE
Please turn off freifunk.net in Baumschulenweg

### ISSUE THREAD

**Issue by zenhund74** [freifunk-berlin/berlin.freifunk.net#116]
Dear freifunk.net community

I live in Baumschulenweg, 12437 Berlin and the freifunk.net doesn`t seem to be functional anymore.
But also, due to it`s elevated strength it competes with normal wifi signals in our house.

I was an absolute fan of this initiative 20 years ago, but I believe it is pretty outdated by now.

Could you just turn it off?

Kind regards, Jens.

**Comment by pktpls** [comment]
Genauer bitte, wir haben keine zentrale Liste aller Standorte

**Comment by xa-y-ax** [comment]
ja. Bitte genaueren Standort. Ich hab Vaterhaus, Rumba und Behringstr. gecheckt. Da scheint alles zu funktionieren.

**Comment by xa-y-ax** [comment]
Can you tell us where the broken Freifunk node is located? It would be ideal if you could see who operates the node. This can be found on the info page at https://frei.funk when you are connected to the Freifunk network.
